tasks/deploy/hyperdrive.ts

```diff
@@ -23,7 +23,8 @@ HyperdriveDeployBaseTask(
 
         // deploy the registry
         await run("deploy:registry", {
-            name: `${network.name.toUpperCase()}_REGISTRY`,
+            // TODO: Generalize this.
+            name: "ElementDAO Hyperdrive Registry",
```

Comment:
why elementdao? this is our registry?  why not delv Hyperdrive registry?

Reply:
Yeah, that's fair. I'll change it.